argocd/structure_application.go

```diff
@@ -284,9 +285,16 @@ func expandApplicationSyncPolicy(_sp []interface{}) (*application.SyncPolicy, er
 			r := _retry[0]
 			for k, v := range r.(map[string]interface{}) {
 				if k == "limit" {
+					var err error
```

Comment:
Is this change related to error shadowing? Please detail.

Reply:
It doesn't seem to build without it either. The `error` var isn't defined earlier, and we can't use `:=` on the next line.